### ISSUE TITLE
The Frenzyheart tribe/ The oracles reputation switch quest

### ISSUE THREAD

**Issue by hardcarry420** [dalaranwow/dalaran-wow#5531]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I've accidentally chosen the faction frenzyheart tribe in scholazar basin when i wanted to be friendly with the oracles. Luckily, there's a solution to this where if you kill Artruis the heartless, you can pick up a quest that let's you change your faction. Sounds great right? Except the quest doesn't work. 
- When you kill Artruis, one of two npc's are going to appear with a quest for you. Now depending on which of the two you leave to live (i choose the oracle murloc since i wanted to become friendly with them) they should give you a quest that you can turn in, and turning in the quest will reverse your reputation with each faction.
Now the problem is, the npc that you're supposed to turn in the quest at, doesn't seem to work properly, cause even after doing the quest, he remains hostile and I can't turn in the quest.

**Expected behaviour**: Tell us what should happen instead

The npc that lets you turn in the quest (High-Oracle Soo-Say) should become friendly, giving you the opportunity to turn in the quest "Return of the Dry Skin", and thus reversing your reputation with the frenzyheart tribe and The oracles. Instead, he remains hostile and you can't turn in the quest. 
**Steps to reproduce the problem**:

1. I've tried abandoning the quest multiple times
2. realizing it works as a daily, I've tried waiting for the daily quest reset before picking up the quest and doing it again
3. ???? all out of ideas at this point. 
**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

https://www.youtube.com/watch?v=LaQ0YQOj4JA - Here's a video from youtube, showing what should happen after you kill Artruis the Heartless

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
https://wowgaming.altervista.org/aowow/?npc=28659 Artruis the Heartless 
https://wowgaming.altervista.org/aowow/?npc=28216 - This is the npc you must kill in order to unlock the quest

https://wowgaming.altervista.org/aowow/?npc=28027 This is the npc that lets you turn in the quest

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/ 

**Include Screenshots from the issue if necessary**

![wowscrnshot_062718_123611](https://user-images.githubusercontent.com/40629989/41969973-6e6d9f08-7a09-11e8-8dce-72a97c2de07d.jpg)

**Comment by galgnu** [comment]
I have achivement from being in exalted with both factions - so it works.

**Comment by galgnu** [comment]
Please read first comment here:
http://www.wowhead.com/npc=28659/artruis-the-heartless#comments

**Comment by alexuktc** [comment]
I did this over a year ago on my toon (get the achievement for both factions) but I had a GM complete the quest for me after a report here was not getting answered. It might be fixed now, but at the time I needed it done it wasn't working properly because there was a quest missing in the q chain. But you could talk to the living npc and it would give a quest to complete. It would just miss the quest after that.

**Comment by Metalshop** [comment]
I am wondering if it is as easy as making sure your not at war under the rep tab?

**Comment by Oreodog** [comment]
It works, did this a couple days ago.

**Comment by Golganeth** [comment]
This quest chain has always worked as intended. I've tested it right now just in case.

Closing
